### ISSUE TITLE
ICSA-23-075-06: Invalid URIs

### ISSUE THREAD

**Issue by tschmidtb51** [cisagov/CSAF#90]
# 🐛 Summary #

ICSA-23-075-06 contains in `/vulnerabilities[]/remediations[4]/url` and `/vulnerabilities[]/remediations[8]/url` the string ` http://process.honeywell.com%20/`. If I'm not completely wrong, this is not a valid URL (as it contains the `%20` as part of the domain name).

## To reproduce ##

1. Get the ICSA-23-075-06](https://raw.githubusercontent.com/cisagov/CSAF/50a6a8aa45d766db17d9c84763b00902a8ae219e/csaf_files/OT/white/2023/icsa-23-075-06.json).
2. Use the [csaf-validator](https://github.com/csaf-poc/csaf_distribution?tab=readme-ov-file#csaf_validator) to check the file.

## Expected behavior ##

The file should conform the schema.

## Any helpful log output or screenshots ##

Paste the results here:

```console
$ csaf_validator csaf_files/OT/white/2023/icsa-23-075-06.json
schema validation errors of "csaf_files/OT/white/2023/icsa-23-075-06.json"
  * https://docs.oasis-open.org/csaf/csaf/v2.0/csaf_json_schema.json#: doesn't validate with https://docs.oasis-open.org/csaf/csaf/v2.0/csaf_json_schema.json#
  * /vulnerabilities/0/remediations/4/url: 'http://process.honeywell.com%20/' is not valid 'uri'
  * /vulnerabilities/0/remediations/8/url: 'http://process.honeywell.com%20/' is not valid 'uri'
  * /vulnerabilities/1/remediations/4/url: 'http://process.honeywell.com%20/' is not valid 'uri'
  * /vulnerabilities/1/remediations/8/url: 'http://process.honeywell.com%20/' is not valid 'uri'
  * /vulnerabilities/2/remediations/4/url: 'http://process.honeywell.com%20/' is not valid 'uri'
  * /vulnerabilities/2/remediations/8/url: 'http://process.honeywell.com%20/' is not valid 'uri'
```

**Comment by mstrad** [comment]
Resolved as of commit ee9bde31389fb3e34482c3efd8d2de7ab0c0ef2b

Thank you for sharing this issue with us.

**Comment by tschmidtb51** [comment]
Unfortunately, the change missed to update the `version` and add a `revision_history` element.
